docs/tutorials/first_steps/06_create_workflow.rst

```diff
@@ -36,33 +35,36 @@ the changes or committing them to the repository.
 
     .. code-block:: console
 
-        git checkout .
-        git clean -fd
+        $ git checkout .
+        $ git clean -fd
 
     Or, if you want to keep your changes, commit with:
 
     .. code-block:: console
 
-        git add -A
-        git commit -m "My own changes"
-        git push
+        $ renku save -m "My own changes"
 
-The ``00-FilterFlights.py`` script takes two input parameters: 1. a file to
+The ``filter_flights.py`` script takes two input parameters: 1. a file to
 process as an input 2. a path for storing the output. So to run it, we would
 normally execute the following:
 
 .. code-block:: console
 
-    mkdir -p data/output        # Create the output directory
-    python src/00-FilterFlights.py data/flight-data/2019-01-flights.csv.zip data/output/2019-01-flights-filtered.csv
+    # Create the output directory
```

Comment:
The "copybutton" should be turned off here. That would make it a little harder to accidentally copy/paste this example, which is not the intent.

Reply:
looks like selectively turning off the copy button is not really possible: https://github.com/executablebooks/sphinx-copybutton/issues/117

Maybe we can just comment those lines? It will look ugly but probably better than people accidentally running those commands

Reply:
I tried various things and I can't really make it any better without making it a comment (which I think is ugly). So I added another warning just before.